php/src/vaas/Options/ForStreamOptions.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace VaasSdk\Options;
+
+class ForStreamOptions
+{
+    const DEFAULT_TIMEOUT = 300;
+    const DEFAULT_REQUEST_ID = null;
```

Comment:
Consider restricting the visibility of these, or is it relevant to users that these defaults are known?